config/config.go

```diff
@@ -176,6 +176,9 @@ type Config struct {
 	// EnableTCP4Only enables net.Listen("tcp4",...)
 	// Note that: it can make lvs with toa work and thus tidb can get real client ip.
 	EnableTCP4Only bool `toml:"enable-tcp4-only" json:"enable-tcp4-only"`
+	// The client will forward the requests to the PD or allocator leader through the follower
```

Comment:
The comment should be updated to be more generic as it doesn't just control the behavior of PD client.

Reply:
Oh, I forgot about it.